01-CORE-SYSTEMS/02-QUALITY_ASSURANCE/bug_catchers/log_analyzer_alerting.py

```
@@ -0,0 +1,833 @@
+                self.config.get('smtp_port', 587)
+            ) as server:
+                if self.config.get('use_tls', True):
+                    server.starttls()
+                
+                if self.config.get('username') and self.config.get('password'):
+                    server.login(self.config['username'], self.config['password'])
+                
+                server.send_message(msg)
+            
+            self.logger.info(f"Email notification sent for alert {alert.alert_id}")
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)